core/backend/src/IModelHost.ts

```diff
@@ -487,7 +487,7 @@ export class IModelHost {
     [
       IModelReadRpcImpl,
       IModelTileRpcImpl,
-      SnapshotIModelRpcImpl,
+      SnapshotIModelRpcImpl, // eslint-disable-line deprecation/deprecation
       WipRpcImpl,
```

Comment:
Didnt realize we actually used this WipRpc...
not in this PR's context but we should review its usage and look at killing it off

Reply:
`WipRpcInterface` has no usage in core and shouldn't be used outside core as it's internal.